haystack/pipelines/utils.py

```diff
@@ -297,18 +307,27 @@ def _format_wrong_examples_report(
         "document_id", "context", "document_id_and_context", "document_id_or_context", "answer", "document_id_or_answer"
     ] = "document_id_or_answer",
     answer_scope: Literal["any", "context", "document_id", "document_id_and_context"] = "any",
+    wrong_examples_fields: List[str] = None,
+    max_chars: int = None,
 ):
+    fields = (
+        ["gold_answers", "answer", "context", "gold_contexts", "document_id", "gold_document_ids"]
```

Comment:
Sorry, I forgot that we print gold_answers and gold_document_ids outside of the predictions. So they are not needed. But let's use this array (without the two) as default values of `wrong_examples_fields` instead. That's way easier to grasp for users.